shell/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -87,80 +93,108 @@
   private static final List<ValidationTask> MASTER_TASKS = new ArrayList<>();
   private static final List<ValidationTask> WORKER_TASKS = new ArrayList<>();
 
-  private static final AlluxioConfiguration CONF;
+  private final AlluxioConfiguration mConf;
+  private final String mPath;
 
-  static {
-    CONF = InstancedConfiguration.defaults();
+  /**
+   * Initializes from the target UFS path and configurations.
+   *
+   * @param path the UFS path
+   * @param conf the UFS configurtions
+   * */
+  public ValidateEnv(String path, AlluxioConfiguration conf) {
```

Comment:
Why we choose to put the validateHdfs logic inside validateEnv?
personally i will prefer to have a separate class for the hdfs<>ValidationTask. and ValidateEnv can call the hdfs<>ValidationTasks if it's root ufs is hdfs. This is because many of the ValidateEnv tasks are not related to hdfs.

Reply:
Actually I tried to move that into `ValidateHdfsMount` but turned out to be very hard. It's hard because the UfsContractTest is not providing a "functional" interface where you just feed it parameters and get results. It has much internal logic and using its embedded fields. Another reason for adding a method to UfsContractTest is, actually most of its tests are run. 

Left a TODO tho. I will have to refactor much of that class to achieve this.